pgctl/config.py

```diff
@@ -127,6 +131,7 @@ def combined(self, defaults=(), args=Dummy()):
             self.from_homedir(),
             self.from_app(),
             self.from_environ(),
+            self.from_config_arg(args),
             self.from_cli(args),
```

Comment:
hmmm, `from_cli` and `from_config_args` sound suspiciously similar -- perhaps they can be combined to avoid confusion?

Reply:
I wasn't sure if it was reasonable to do that given the fact that Config.combine() merges each top-level config type, but I maybe it's better after all.